db/flag.go

```diff
@@ -21,17 +22,17 @@ func GetDefaultFlags() Flags {
 }
 
 type Flag struct {
-	Address     string
-	TwitterName string
-	Flags       Flags
+	Address string
+	UserID  string
```

Comment:
Same as `AddressLInkedKey`, why not `int64`? Also the rest of db items that use `UserId`